docs/standards/smart-contracts/lsp0-erc725-account.md

```diff
@@ -102,6 +106,12 @@ Executed when sending bytes data to the contract and the first 4 bytes of this d
 
 2.Then return the return value received after this low level call.
 
+- If the data sent is **preprended with 4 zeros (0)**, then the call should be checked for an extension, if there is none the call should still pass.
+
+  For example, when sending value to the contract it can be associated with a message, such as `"Here is 1 Ether"`, after encoding the message as bytes: `0x486572652069732031204574686572`.
```

Comment:
Suggest breaking it into a separate paragraph to make the _example_ section more visually noticeable:

```suggestion
_Example:_ 

When sending value to the contract with a message alongside the value transfer, such as `"Here is 1 Ether"`. The message encoded as bytes will be: `0x486572652069732031204574686572`.
```